server/backend/testspg/crud/test_idim_proxy_service.py

```diff
@@ -14,16 +14,24 @@ class TestIdimProxyServiceClass(object):
     """
     Testing IdimProxyService class with real remote API calls (TEST environment).
     """
-    search_params = IdimProxySearchParamIdir(**{"userId": "ianliu"}) # Valid test IDIR user.
+
+    TEST_BUSINESS_BCEID_USERNAME = "LOAD-2-TEST"
+    search_params = IdimProxySearchParam(
+        **{"userId": "ianliu"}
+    )  # Valid test IDIR user.
+    search_params_business_bceid = IdimProxySearchParam(
+        **{"userId": TEST_BUSINESS_BCEID_USERNAME}
+    )  # Valid test Business Bceid user.
 
     def setup_class(self):
         # local valid mock requester
         self.requester = Requester(
-            ** {
+            **{
                 "cognito_user_id": "dev-idir_e72a12c916a44f39e5dcdffae7@idir",
                 "user_name": "IANLIU",
-                "user_type": "I",
-                "access_roles": ["FAM_ADMIN", "FOM_DEV_ADMIN"]
+                "user_type_code": "I",
+                "access_roles": ["FAM_ADMIN", "FOM_DEV_ADMIN"],
+                "user_guid": os.environ.get("TEST_IDIR_USER_GUID"),
```

Comment:
currently since we need to use IDIR guid, so set it through env vars when run locally